filters_ublock.txt

```diff
@@ -39,10 +39,9 @@ search.seznam.cz,clanky.seznam.cz##div[data-e-b-n*="advert"]
 sreality.cz##+js(nosiif, checkRods)
 super.cz##+js(ra, style, body)
 svetandroida.cz##+js(aopr, detectAdBlocker)
-titulky.com##+js(ra, style, div[id^="head"])
-titulky.com##+js(set, showBanner600, true)
-titulky.com##+js(aeld, load)
-titulky.com##+js(aopr, onload)
+titulky.com##+js(set, showBanner700, true)
+titulky.com##+js(set, showFairUser, true)
+titulky.com##+js(nano-stb, CountDown)
```

Comment:
Který countdown to ovlivňuje?

Reply:
Toto ovplyvnuje countown v popupe na stahovanie titulkov pre neprihlasenych pouzivatelov. Defaultne tam je 10s delay pokym sa zobrazi download button.

Reply:
tu je konkretna funkcia ktoru to ovplyvnuje -> `https://www.titulky.com/downfnc.js?2`

Reply:
Rychleji to zobrazí ten download button ale nejde stáhnout dřív.

Reply:
otestoval som to - mas pravdu, aj ked sa download btn zobrazi skorsie, tak download funguje az po countdowne. tym padom to nedava moc zmysel, countdown teda vyhodim